### PR TITLE
made cover page responsive

### PR DESCRIPTION
**Pull request by nlok5923**
fixes : #6 
you can see the updated site here : https://youthful-liskov-ec4953.netlify.app/

### CODE REVIEW

**Comment by nlok5923** [comment]
@AmanRaj1608 please review this pull request.

**Review by AmanRaj1608 — CHANGES_REQUESTED** [review]
See the comment

**Comment by AmanRaj1608** [comment]
- Set the home height to full screen using `90vh`.
- Remove the gap between the buttons as per design.
- font-weight not matching as per design. In small screens decrease the font size.
- Also set the navbar color to the same as you have used in the home section.

Current view
image

Expected 
image

**Comment by nlok5923** [comment]
Okay sir I am working on it.

File: src/components/Home/Cover/Cover.css [hunk 1]
```diff
@@ -72,8 +72,8 @@
   color: #000000;
 }
 .coder-image {
-  width: 380px;
-  height: 430px;
+  width: 400px;
+  height: 500px;
 }
 @media only screen and (max-width: 900px) {
```
Comment (AmanRaj1608) [inline src/components/Home/Cover/Cover.css]: The breakpoint is 900px
It doesnt look goods on medium-size device as 
From this 

image

image

**Review by AmanRaj1608 — CHANGES_REQUESTED** [review]
On small devices don't hardcode px values.
It's better to use `vw` , `%` so that it adjusts for many devices instead of hardcoding values.

File: src/components/Home/Cover/Cover.css [hunk 2]
```diff
@@ -72,8 +72,8 @@
   color: #000000;
 }
 .coder-image {
-  width: 380px;
-  height: 430px;
+  width: 400px;
```
[the remark this answers is not in the document]
Comment (nlok5923) [reply src/components/Home/Cover/Cover.css]: sir 490&596 will be too big i think .
Screenshot (1347)

**Comment by nlok5923** [comment]
@AmanRaj1608 sir please review the changes.

File: src/components/Home/Cover/Cover.css [hunk 3]
```diff
@@ -1,48 +1,57 @@
-@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Open+Sans:wght@300&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Montserrat&family=Noto+Sans&family=Open+Sans:wght@300&display=swap");
-
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
 #cover-display {
```
Comment (yashshah2820) [inline src/components/Home/Cover/Cover.css:4]: add a link space between add proper formatting wherever needed

**Review by yashshah2820 — CHANGES_REQUESTED** [review]
apply proper formatting between different css classes.

File: src/components/Header/Header.css [hunk 4]
```diff
@@ -1,132 +1,132 @@
...
-  }
-}
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 10%;
+  background-color: #181818;
+}
+@media (max-width: 860px) {
```
Comment (yashshah2820) [inline src/components/Header/Header.css:8]: still not formatted 🤕

File: src/components/Home/Cover/Cover.css [hunk 5]
```diff
@@ -1,48 +1,58 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap');
 
 #cover-display {
   background-color: #181818;
   width: 100%;
+  height: 100vh;
   overflow-x: hidden;
+  overflow-y: hidden;
 }
 #flex-container {
```
Comment (yashshah2820) [inline src/components/Home/Cover/Cover.css:12]: add a line here

File: src/components/Home/Cover/Cover.css [hunk 6]
```diff
@@ -1,48 +1,58 @@
...
   display: flex;
   flex-direction: row;
+  width: 83%;
   justify-content: space-between;
   margin: auto;
+  align-items: center;
 }
 .display-head {
```
Comment (yashshah2820) [inline src/components/Home/Cover/Cover.css:20]: add a line here

**Review by yashshah2820 — CHANGES_REQUESTED** [review]
formatting required in Cover.css

File: src/components/Home/Cover/Cover.css (hunk 6 above)
Comment (nlok5923) [reply src/components/Home/Cover/Cover.css:20]: Sorry sir i thought something different before.

File: src/components/Header/Header.css [hunk 7]
```diff
@@ -1,132 +1,132 @@
...
+  font-size: 36px;
+  display: block;
+  transition: all 0.3s ease 0s;
+}
+
+.overlay a:hover,
+.overlay a:focus {
+  color: #0088a9;
+}
+.overlay .close {
```
Comment (yashshah2820) [inline src/components/Header/Header.css:105]: add a line here

File: src/components/Home/Cover/Cover.css [hunk 8]
```diff
@@ -1,70 +1,136 @@
...
+  align-items: center;
 }
+
 .display-head {
   display: flex;
   flex-direction: column;
   justify-items: center;
-  margin-left: 140px;
 }
 .display-heading {
```
Comment (yashshah2820) [inline src/components/Home/Cover/Cover.css:27]: add a line here

File: src/components/Home/Cover/Cover.css [hunk 9]
```diff
@@ -1,70 +1,136 @@
...
   margin-top: 77px;
-  font-weight: 300;
+  font-weight: 50;
   font-size: 64px;
+  font-size-adjust: 0.58;
   line-height: 78px;
   letter-spacing: -0.05em;
   color: #ffffff;
 }
+.manage-goodline {
```
Comment (yashshah2820) [inline src/components/Home/Cover/Cover.css:38]: add a line here

File: src/components/Home/Cover/Cover.css [hunk 10]
```diff
@@ -1,70 +1,136 @@
...
+  width: 80%;
+}
 .good-line {
```
Comment (yashshah2820) [inline src/components/Home/Cover/Cover.css:41]: add a line here

File: src/components/Home/Cover/Cover.css [hunk 11]
```diff
@@ -1,70 +1,136 @@
...
   font-weight: 300;
-  font-size: 36px;
-  line-height: 49px;
-  letter-spacing: -0.05em;
+  font-size: 32px;
+  line-height: 45px;
+  letter-spacing: -0.04em;
   color: #ffffff;
 }
 .inf-btn {
```
Comment (yashshah2820) [inline src/components/Home/Cover/Cover.css:50]: add a line here

File: src/components/Home/Cover/Cover.css [hunk 12]
```diff
@@ -1,70 +1,136 @@
...
   display: flex;
   flex-direction: row;
   justify-content: space-between;
   margin-top: 65px;
+  margin-left: 10%;
+  margin-right: 20%;
 }
 .trans-btn {
```
Comment (yashshah2820) [inline src/components/Home/Cover/Cover.css:58]: add a line here

File: src/components/Home/Cover/Cover.css [hunk 13]
```diff
@@ -1,70 +1,136 @@
...
   line-height: 33px;
   letter-spacing: -0.05em;
   color: #f638dc;
-  padding: 5px 20px;
+  padding: 1% 9%;
   background-color: #181818;
   cursor: pointer;
 }
-.trans-btn:hover{
+.trans-btn:hover {
```
Comment (yashshah2820) [inline src/components/Home/Cover/Cover.css:73]: add a line here

File: src/components/Home/Cover/Cover.css [hunk 14]
```diff
@@ -1,70 +1,136 @@
...
   color: #000000;
 }
 .coder-image {
-  margin-right: 117px;
-  width: 450px;
-  height: 550px;
+  width: 40%;
+  height: 55%;
+}
+@media only screen and (max-width: 1050px) {
```
Comment (yashshah2820) [inline src/components/Home/Cover/Cover.css:81]: add a line here

**Review by yashshah2820 — COMMENTED** [review]
still not formatted, take a look 😞

**Review by AmanRaj1608 — APPROVED** [review]

**Pull request merged**